Dockerfile

```diff
@@ -17,16 +17,14 @@ COPY go.sum ./
 RUN go mod download
 COPY . .
 
-RUN apk update && apk add bash git openssh
+RUN apk --no-cache add bash git openssh
```

Comment:
This line should be moved above `COPY go.mod ./`, because when a step’s cache is invalidated by changing files or modifying lines in the Dockerfile, subsequent steps of their cache will break. So order the steps from least to most frequently changing steps to optimize caching.

Reply:
fixed in fee1a31da521fd02821b509be37e72d0c975537c